### PR TITLE
Fix OpenGL rendering on Kodi Leia 

### PR DESCRIPTION
**Pull request by AlwinEsch**
Like on the others fix this the rendering, further can be https://github.com/xbmc/screensaver.stars/issues/3 and https://trac.kodi.tv/ticket/16443 closed, with https://github.com/xbmc/screensaver.stars/issues/7 I'm not sure, there noted for Krypton.

bildschirmfoto von 2018-12-01 10-56-47

Also added Fanart and images:
bildschirmfoto von 2018-12-01 15-37-25

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1 +1 @@
-buildPlugin(platforms: ['windows-i686', 'windows-x86_64'])
+buildPlugin()
```
Comment (Rechi) [inline Jenkinsfile]: tested on android-aarch64, doesn't work

Comment (AlwinEsch) [reply Jenkinsfile]: You mean compile or usage? The compile becomes fixed.

Comment (Rechi) [reply Jenkinsfile]: usage, it already compiles

Comment (AlwinEsch) [reply Jenkinsfile]: Have you a kodi.log, maybe the shader load brings a error message.

Comment (AlwinEsch) [reply Jenkinsfile]: If yes maybe related with here https://github.com/xbmc/screensaver.asteroids/tree/master/screensaver.asteroids/resources/shaders and I bring in the separated shaders for GL and GLES

**Comment by AlwinEsch** [comment]
Is updated and testet on Linux, Android and Windows.

**Pull request merged**